slices/libldap2.yaml

```diff
@@ -0,0 +1,18 @@
+package: libldap2
+
+essential:
+  - libldap2_copyright
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libgnutls30_libs
+      - libsasl2-2_libs
+    contents:
+      /usr/lib/*-linux-*/liblber.so.2*:
+      /usr/lib/*-linux-*/libldap.so.2*:
+
+  copyright:
+    contents:
+      /usr/share/doc/libhogweed6/copyright:
```

Comment:
This should be `/usr/share/doc/libldap2/copyright:`.

Reply:
changed